src/Symfony/Component/Security/Http/Authenticator/Passport/Passport.php

```diff
@@ -68,9 +68,17 @@ public function getUser(): UserInterface
      *
      * @return $this
```

Comment:
```suggestion
     * @param string|null $badgeFqcn A FQCN to which the badge should be mapped to. 
     *                                                        This allows replacing a built-in badge by a custom one using
     *.                                                        e.g. addBadge(new MyCustomUserBadge(), UserBadge::class)
     *
     * @return $this
```

This will also make DebugClassLoader able to trigger a notice in case a custom Passport does not implement the new parameter

Reply:
Good for me, but fails `fabpot` CI cause the parameter does not really exist